script/import_scripts/base.rb

```diff
@@ -198,12 +198,7 @@ def all_records_exist?(type, import_ids)
     return false if import_ids.empty?
 
     existing = "#{type.to_s.classify}CustomField".constantize.where(name: 'import_id')
-
-    if Fixnum === import_ids.first
-      existing = existing.where('cast(value as int) in (?)', import_ids)
-    else
-      existing = existing.where('value in (?)', import_ids)
-    end
+    existing = existing.where('value in (?)', import_ids.map(&:to_s))
```

Comment:
@riking I had to remove the cast. Some importers (like phpBB3) insert numbers and strings as `import_id` into the same custom_fields table. This change prevents cast errors during imports.